tests/fixtures/analysis_form/v1.json

```diff
@@ -61,11 +61,15 @@
             "en": {
               "value": "Hello how may I help you?"
             }
-          },
-          "acme_timestamp": "2021-11-01Z"
+          }
         },
         "clerk_details-name_of_clerk": {
-          "comment": "Sounds like an interesting person"
+          "qual": [
+            {
+              "uuid": "comment",
+              "val": "Sounds like an interesting person"
+            }
+          ]
         }
```

Comment:
To match the new schema in KPI; obviously, "comment" is not a UUID, but perhaps it's useful for debugging? I'm open to making these look more UUID-ish—but a true, random UUIDs probably aren't that helpful